interval/bounds.go

```diff
@@ -0,0 +1,54 @@
+package interval
+
+import (
+	"fmt"
+	"math"
+
+	"github.com/influxdata/flux/values"
+)
+
+const (
+	MaxTime = math.MaxInt64
+	MinTime = math.MinInt64
+)
+
+type Bounds struct {
+	start values.Time
+	stop  values.Time
+	index int
```

Comment:
How is the index being used and can we document its purpose?

Reply:
@jsternberg does this comment help clarify `index` for you?